content/docs/0.6.0/installation/backup_and_restore/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Backup and restore Keptn
+description: How to backup and restore Keptn.
+weight: 20
+keywords: backup
+---
+
+To secure all your data in your project's git repositories, as well as all events that have occurred for these project, you need to 
+back up the data within the Configuration Service, the MongoDB and - if you have configured those - credentials to your project's git upstream repos.
+The following sections describe how to back up that data and store it on your local machine.
+**NOTE:** These instructions do not cover backing up any Helm- or Istio-related configurations.
+
+## Backup Configuration Service
+
+1. First, create a local backup directory containing all data within the git repos of the configuration-service. To do so, please execute the following command:
+
+```console
+mkdir keptn-backup
+cd keptn-backup
+mkdir config-svc-backup
+CONFIG_SERVICE_POD=$(kubectl get pods -n keptn -lrun=configuration-service -ojsonpath='{.items[0].metadata.name}')
+kubectl cp keptn/$CONFIG_SERVICE_POD:/data ./config-svc-backup/ -c configuration-service
+```
+
+1. Verify that the data has been copied correctly to your local machine by checking the content of the directory you just created. 
```

Comment:
The enumeration with `1.` does not seem to work.

I think there should not be a newline between the two lists. Anyway, you can use `2.` here too.